base/notification/notification.go

```diff
@@ -73,8 +74,13 @@ type Advisory struct {
 	Synopsis     string `json:"synopsis"`
 }
 
-func MakeNotification(system *models.SystemPlatform, event *mqueue.PlatformEvent,
-	eventType string, events []Event) *Notification {
+func MakeNotification(system *models.SystemPlatform, event *mqueue.PlatformEvent, eventType string, events []Event) (*Notification, error) {
+
+	orgID := event.GetOrgID()
+	if orgID == "" || orgID == "null" {
```

Comment:
I see notifications backend receives this value and we just pass it directly from Kafka message. It might be they set it up themselves but I wanted to be safe here in case it came from Inventory.